pkg/scheduler/plugins/overcommit/overcommit.go

```diff
@@ -88,10 +88,17 @@ func (op *overcommitPlugin) OnSessionOpen(ssn *framework.Session) {
 	}
 	op.idleResource = total.Clone().Multi(op.overCommitFactor).Sub(used)
 
-	// calculate inqueue job resources
 	for _, job := range ssn.Jobs {
+		// calculate inqueue job resources
 		if job.PodGroup.Status.Phase == scheduling.PodGroupInqueue && job.PodGroup.Spec.MinResources != nil {
 			op.inqueueResource.Add(api.NewResource(*job.PodGroup.Spec.MinResources))
```

Comment:
need to consider below conner case while PodGroup is in `Inqueue`. 
driver pod is running, part or all of executor pod are pending. The already allocated resource might need to be excluded from op.inqueueResource  and then compare with `idle` resource.

Reply:
> need to consider below conner case while PodGroup is in `Inqueue`. driver pod is running, part or all of executor pod are pending. The already allocated resource might need to be excluded from op.inqueueResource and then compare with `idle` resource.

It will not be at `inqueue` phase for Spark application because it will be turn into `running` at the end of the first session. The default `minMember` will be 1 and the allocated pod number is at least 1.